source/redirects.txt

```diff
@@ -74,6 +79,11 @@ Rules
     For example, ``[v1.0-v2.0)`` would match ``v1.0`` and ``v1.5``, but **NOT**
     ``v2.0``.
 
+.. important::
+
+   Include the trailing slash (``/``) on both the source and target when
```

Comment:
@schmalliso are they required on the source? I looked at https://github.com/mongodb/docs-bi-connector/commit/19aa9023e06065b3c1bcbc76dd37d84f3498910f for reference and it seems like it's required on the target, but inconsistent on the source?

Should we just recommend it anyway?